src/components/ChatPanel.tsx

```diff
@@ -74,6 +74,8 @@ export interface ChatPanelProps
    * can click on instead of typing their own.
    */
   messageSuggestions?: string[];
+  /** A callback which is called when user clicks a footer link. */
```

Comment:
shouldn't be just footer link. It should be any link present in Panel in general, which include the MessageBubble and potentially MessageSuggestions.

Reply:
ah ok! I added onLinkClick to MessageBubble as well, but I don't think we can have clickable link within MessageSuggestions (clicking the suggestion = sending the whole msg).